### DISCUSSION TITLE
Set entity sensor to unavailable / unknown

### DISCUSSION THREAD

**Discussion by dflvunoooooo** [zachowj/node-red-contrib-home-assistant-websocket#1044] [Q&A] [answered]
How do I set a numeric entity sensor to unknown / unavailable? I didn't find anything in the docs about this. I have a sensor for the internet speed, which is updated every 10 minutes. If it doesn't get an update after 15 minutes, it should change to unknown. It might be related to this feature request [520](https://github.com/zachowj/node-red-contrib-home-assistant-websocket/discussions/520).

It was suggested to me [here](https://community.home-assistant.io/t/set-entity-sensor-to-unavailable-unknown/608302) to add an attribute "available" und set this with an input to false. But if I do so, Node-Red complains about a missing state in the message. But I obviously do not want to change the state.

What I do till now, is to change the state to "unknown". But this way home assistant throws errors, complaining about a non numerical state in a numerical sensor.

**Comment by zachowj** [comment, marked as answer]
Currently, this is not possible.

**Reply by dflvunoooooo** [reply]
That really is a pity. This results in two error messages in the home assistant log.
